### ISSUE TITLE
Change of wanted_words resulting very low recognition accuracy (only 24%)

### ISSUE THREAD

**Issue by auvilink** [xiongyihui/ML-KWS-for-MCU#2]
Hi, xiongyihui

 I changed several wanted_words in "train.py" as:
"up,down,left,right,stop,go,follow,forward,backward,on,off"
All the words / .wav files are in original speech commands dataset V_0.0.2.
I did not use my own wav files.

I then trained ds_cnn_s model using instructions in "train_commands.txt"
"python train.py --model_architecture ds_cnn --model_size_info 5 64 10 4 2 2 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 --dct_coefficient_count 10 --window_size_ms 40 --window_stride_ms 20 --learning_rate 0.0005,0.0001,0.00002 --how_many_training_steps 10000,10000,10000 --summaries_dir work/DS_CNN/DS_CNN1/retrain_logs --train_dir work/DS_CNN/DS_CNN1/training"

I changed the wanted_words in "test.py" to the same ones above and

"python test.py --model_architecture ds_cnn --model_size_info 5 64 10 4 2 2 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 64 3 3 1 1 --checkpoint=/Users/michaelhe/ml/asr/ML-KWS-for-MCU/work/DS_CNN/DS_CNN1/training/best/ds_cnn_9294.ckpt-27200"

INFO:tensorflow:set_size=33946
INFO:tensorflow:Confusion Matrix:
[[2837 0 0 0 0 0 0 0 0 0 0 0 0]
[ 60 2174 0 0 0 0 0 654 0 0 0 0 0]
[ 163 1902 22 0 0 0 0 800 0 0 0 3 0]
[ 48 1225 0 35 0 0 0 1909 0 0 0 0 0]
[ 52 2636 0 0 12 7 0 409 0 0 0 0 0]
[ 36 2420 0 0 0 345 0 180 0 0 0 0 0]
[ 86 1204 0 0 0 0 0 1750 0 0 0 0 0]
[ 34 190 0 0 0 0 0 2884 0 0 0 0 0]
[ 42 811 0 0 0 0 0 364 72 0 0 0 0]
[ 23 989 0 0 0 0 0 181 4 1 0 0 0]
[ 17 1257 0 0 0 0 0 98 0 0 0 0 0]
[ 55 2400 0 0 0 0 0 486 0 0 0 136 0]
[ 122 1668 0 0 0 0 0 1141 0 0 0 1 1]]
INFO:tensorflow:Training accuracy = 25.10% (N=33946)
INFO:tensorflow:set_size=3999
INFO:tensorflow:Confusion Matrix:
[[334 0 0 0 0 0 0 0 0 0 0 0 0]
[ 4 251 0 0 0 0 0 79 0 0 0 0 0]
[ 19 220 1 0 0 0 0 110 0 0 0 0 0]
[ 3 102 0 4 0 0 0 268 0 0 0 0 0]
[ 6 304 0 0 1 1 0 40 0 0 0 0 0]
[ 4 302 0 0 0 36 0 21 0 0 0 0 0]
[ 12 128 0 0 0 0 0 210 0 0 0 0 0]
[ 3 30 0 0 0 0 0 339 0 0 0 0 0]
[ 1 80 0 0 0 0 0 44 7 0 0 0 0]
[ 0 118 0 0 0 0 0 27 0 1 0 0 0]
[ 3 140 0 0 0 0 0 10 0 0 0 0 0]
[ 7 272 0 0 0 0 0 75 0 0 0 9 0]
[ 13 209 1 0 0 0 0 149 0 0 0 1 0]]
INFO:tensorflow:Validation accuracy = 24.58% (N=3999)
INFO:tensorflow:set_size=4492
INFO:tensorflow:Confusion Matrix:
[[375 0 0 0 0 0 0 0 0 0 0 0 0]
[ 5 300 0 0 0 1 0 69 0 0 0 0 0]
[ 25 274 2 0 0 0 0 124 0 0 0 0 0]
[ 6 160 0 4 0 0 0 236 0 0 0 0 0]
[ 5 352 0 0 0 0 0 55 0 0 0 0 0]
[ 3 331 0 0 0 20 0 42 0 0 0 0 0]
[ 11 165 0 0 0 0 0 235 0 0 0 0 0]
[ 5 28 0 0 0 0 0 369 0 0 0 0 0]
[ 2 124 0 0 0 0 0 40 6 0 0 0 0]
[ 3 135 0 0 0 0 0 16 1 0 0 0 0]
[ 2 146 0 0 0 0 0 17 0 0 0 0 0]
[ 9 297 0 0 0 0 0 73 2 0 0 15 0]
[ 13 227 0 0 0 0 0 161 1 0 0 0 0]]

**### **INFO:tensorflow:Test accuracy = 24.29% (N=4492)

24.29% accuracy seems very low.****

### Did I make any mistake somewhere in the process?

Thank you so much.
